src/reminders/Reminder.ts

```diff
@@ -0,0 +1,77 @@
+import type { Moment } from 'moment';
+import { getSettings } from '../Config/Settings';
+
+export class ReminderSettings {
+    notificationTitle: string = 'Task Reminders';
+    dateFormat: string = 'YYYY-MM-DD';
```

Comment:
No other dates allow custom format strings in Tasks.

If we ever add a custom date formatting facility, it should be provided for all dates and not just reminders.

Therefore I would prefer the formatting component of the reminders to only allow customising the time format, and not the date format.

Reply:
There isn't a setting to change the dateFormate. It mostly came about me wanting to keep single constant for 'YYYY-MM-DD' and having it accessible the same object as dateTimeFormat seemed like a good option. Open to a different solution, maybe a special file for tasks constants?

Reply:
Solving this since another thread is already open for this https://github.com/obsidian-tasks-group/obsidian-tasks/pull/1925#discussion_r1181465042